pom.xml

```diff
@@ -142,5 +143,24 @@
         <module>build/optaplanner-distribution-internal</module>
       </modules>
     </profile>
+    <profile>
+      <id>jakartaMigration</id>
```

Comment:
I'd consider calling this something more general, such as "migration8to9", or some such.